### PR TITLE
webui logpath and document

### PR DESCRIPTION
**Pull request by lvybriage**
1. update ant design version
2. add webui document
3. make logpath is a clickable hyperlink when it's http or https
4. add feedback button

### CODE REVIEW

File: docs/WebUI.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# WebUI
+
+NNI is a research platform for metalearning. It provides easy-to-use interface so that you could perform neural architecture search, hyperparameter optimization and optimizer design for your own problems and models.
```
Comment (chicm-ms) [inline docs/WebUI.md]: The description of NNI should have already been somewhere else, we do not need to repeat it.

File: docs/WebUI.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Web UI allows user to monitor the status of the NNI system using a graphical interface.
+
+## Deployment
+
+### To start the webui
+
+* cd webui
```
Comment (chicm-ms) [inline docs/WebUI.md]: webui is shipped within NNI package and is started by nnictl command.  we do not need to expose this yarn start method for end user.

File: src/webui/README.md [hunk 3]
```diff
@@ -7,7 +7,14 @@ Web UI allows user to monitor the status of the NNI system using a graphical int
 
 ### To start the webui
 
+* cd webui
+
 > $ yarn
+
+> $ yarn build
```
Comment (chicm-ms) [inline src/webui/README.md]: webui is started by nnictl command.

File: src/webui/package.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "antd": "^3.8.1",
+    "antd": "^3.9.3",
```
Comment (chicm-ms) [inline src/webui/package.json]: Is this antd upgrade neccessary?

Comment (lvybriage) [reply src/webui/package.json]: 是的。我用的一些新版本的图标。

**Pull request merged**